prov/verbs/src/verbs_eq.c

```diff
@@ -33,6 +33,7 @@
 
 #include "config.h"
 
+#include <infiniband/cm.h>	/* For Reject Codes */
```

Comment:
The Intel CI failure is because this header is not available.  I don't know that it usually is.  The IB CM reject code is part of the IB standard, so we can provide our own definition to the correct value.

Reply:
Thank you, I'll update to define it and nix the header.

Reply:
Done. Thanks.